charts/federation-v2/charts/controllermanager/templates/federationconfig.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: core.federation.k8s.io/v1alpha1
+kind: FederationConfig
+metadata:
+  name: federation-v2
+  namespace: {{ .Release.Namespace }}
+spec:
+  limited-scope: {{ .Values.global.limitedScope }}
+{{- if .Values.registryNamespace }}
+  registry-namespace: {{ .Values.registryNamespace }}
+{{- else }}
+{{- if .Values.global.limitedScope }}
+  registry-namespace: {{ .Release.Namespace }}
+{{- else }}
+  registry-namespace: kube-multicluster-public
+{{- end }}
+{{- end }}
+  controller-duration:
+    available-delay: {{ .Values.clusterAvailableDelay | default "20s" | quote }}
```

Comment:
(No action required) I think it's ok to set defaults in this PR, but longer-term I'm not sure that we want to set a value here if none were provided vs having the controller manager supply its own default.  I also wonder if the controller manager should update the config with missing defaults to ensure that it is the sole arbiter of what default value should be.

Reply:
If there is no `FederationConfig`, controller will set the defaults.

Reply:
Given that a value of zero is invalid for any of these values, that's a good indication to avoid setting a value.